features/svg-deprecated.yml

```diff
@@ -1,7 +1,11 @@
-draft_date: 2024-10-085
-name: svg-deprecated
-description: Placeholder for deprecated or non-real features
+name: SVG (deprecated)
+description: SVG 2 deprecated or removed a significant number of features defined in SVG 1.1, including features related to fonts, alternate glyphs, the `xlink` namespace, and the `<cursor>` and `<tref>` elements.
```

Comment:
This was a bit difficult to write, as this is a junk drawer feature, and its entire purpose for existing is to describe that deprecations occurred, and not the features themselves.